raiden/network/transport/matrix/utils.py

```diff
@@ -251,6 +251,22 @@ def _presence_listener(self, event: Dict[str, Any]):
         if self._user_presence_changed_callback:
             self._user_presence_changed_callback(user, new_state)
 
+    def get_status_info(self):
+        user_ids_to_presence_for_known_addresses = {
+            to_checksum_address(address): {
+                user_id: self._userid_to_presence[user_id].value
+                for user_id in self._address_to_userids[address]
+            }
+            for address in self.known_addresses
+        }
+
+        log.debug(
+            "Matrix status info",
+            current_user_id=self._user_id,
+            user_ids_and_presence_for_known_addresses=user_ids_to_presence_for_known_addresses,
```

Comment:
The variable is maybe a bit too detailed ;)

Reply:
ooh not pythonic?:D Changed it `addresses_uids_presence`